tasks/common.yml

```diff
@@ -21,7 +21,7 @@
     notify:
       - restart sensu-server service
       - restart sensu-api service
-      - restart sensu-client service
```

Comment:
What do you think of a new variable `sensu_client` to deploy sensu client? @smuth4